.github/actions/reusable-python-setup_poetry/action.yml

```diff
@@ -30,6 +43,9 @@ runs:
       run: |
         poetry config virtualenvs.create true
         poetry config virtualenvs.in-project true
+        if (${{ inputs.extra_repository}}); then
+          poetry config http-basic.mira-pypi ${{ inputs.repo_user }} ${{ inputs.repo_token }}
```

Comment:
**question:** could it get the secrets directly instead of getting them as workflow input?

Reply:
ok, this is the action. That is fine. I got confused with the workflow